tasks_management/services.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from django.db import transaction
+
+from core.services import create_or_update_core_user, BaseService
+from core.services.utils import check_authentication, output_exception, model_representation, output_result_success
+from tasks_management.models import TaskGroup, TaskExecutor
+from tasks_management.validation import TaskGroupValidation, TaskExecutorValidation
+
+logger = logging.getLogger(__name__)
+
+
+class TaskGroupService:
+    OBJECT_TYPE = TaskGroup
+
+    def __init__(self, user):
+        self.user = user
+        self.validation_class = TaskGroupValidation
+
+    @check_authentication
+    def create(self, obj_data):
```

Comment:
We could add some documentation regarding the object data (typehint and ideally short docstring). As developer I wouldn't like to have to go though the implementation just to know what input data is required.

Reply:
added types